web/migrations/0054_question_choice_response_survey_question_survey.py

```
@@ -0,0 +1,83 @@
+            ],
+        ),
+        migrations.CreateModel(
+            name="Response",
+            fields=[
+                ("id", models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("text_answer", models.TextField(blank=True)),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                (
+                    "choice",
+                    models.ForeignKey(
+                        blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to="web.choice"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)